hikari/stickers.py

```diff
@@ -156,9 +156,15 @@ def image_url(self) -> files.URL:
         """URL for the image.
 
         The extension will be based on `format_type`. If `format_type` is `StickerFormatType.LOTTIE`,
-        then the extension will be `.json`. Otherwise, it will be `.png`.
+        then the extension will be `.json`, if it's `StickerFormatType.GIF` it will be `.gif`. Otherwise, it will be `.png`.
         """
-        ext = "json" if self.format_type is StickerFormatType.LOTTIE else "png"
+        ext = (
+            "json"
+            if self.format_type is StickerFormatType.LOTTIE
+            else "gif"
+            if self.format_type is StickerFormatType.GIF
+            else "png"
+        )
```

Comment:
It will probably be a lot better to use a mapping and then do:

```
_EMOJI_EXTENSIONS.get(self.format_type, "png")
```

Reply:
I was reluctant to create a mapping for just 3 formats, but I could make it happen!

Reply:
Honestly I do not think this is that much better, but there you go :P